wrapper/src/main/java/software/amazon/jdbc/plugin/staledns/AuroraStaleDnsPlugin.java

```diff
@@ -56,6 +56,8 @@ public class AuroraStaleDnsPlugin extends AbstractConnectionPlugin {
   private final PluginService pluginService;
   private final AuroraStaleDnsHelper helper;
 
+  static final String IS_READER_COLUMN = "is_reader";
```

Comment:
Do we still need it here in the  plugin? Should we relocate it to the helper class?